Functions/Quantification/xASL_quant_HadamardDecoding.m

```diff
@@ -42,7 +62,7 @@
     error('xQ input is empty');
 end
 
-%% 1. Specify the Encoding matrix
+%% 1. Specify the decoding matrix
```

Comment:
Sorry. Outdated message. It is indeed NxN. The message is fixed.
While the first line is the same and we could accept N-1 x N and add the first line automatically, everybody is used to provide the NxN version in code and papers and will be easier for the users to use NxN as well. And it will be easier for BIDS as well. 

You can resolve, this message was fixed.